FlowCrypt/Controllers/Compose/ComposeViewController.swift

```diff
@@ -285,6 +290,11 @@ extension ComposeViewController: ASTableDelegate, ASTableDataSource {
                 case .text: return self.textNode(with: nodeHeight)
                 case .subjectDivider: return DividerCellNode()
                 }
+            case (.main, 2):
```

Comment:
I wonder if the whole `tableNode` should be in some sort of decorator class? It looks very UI-heavy with not much business logic. What do you think @Kharchevskyi ?

Reply:
If so, then that would apply to the whole `extension ComposeViewController: ASTableDelegate, ASTableDataSource {` section

Reply:
I agree, we may create DataSource entity for this screen, @Kharchevskyi let's discuss that when it's convenient